community-operators/triggermesh/triggermesh.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,167 @@
+  
+    ### Supported Features:
+    **Knative services**
+    Knative services can be deployed from source manifest or by specifying a container image or by specifying a git repository.
+    **Knative Build**
+    Container builds and more generically workflows can be configured and deployed.
+    **Knative Eventing**
+    Event sources such as GitHub, GitLab and a few AWS event sources can be configured and deployed.
+  displayName: TriggerMesh
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          - serviceaccounts
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - namespaces
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - extensions
+          resources:
+          - ingresses
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - apps
+          resourceNames:
+          - operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterrolebindings
+          - rolebindings
+          verbs:
+          - '*'
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - '*'
+        - apiGroups:
+          - build.knative.dev
+          resources:
+          - buildtemplates
+          verbs:
+          - '*'
+        - apiGroups:
+          - app.triggermesh.io
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: triggermesh-operator
+      deployments:
+      - name: triggermesh-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: triggermesh-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: triggermesh-operator
+            spec:
+              containers:
+              - env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
```

Comment:
Please set the fieldPath to `metadata.annotations['olm.targetNamespaces']` 
This annotation will be added by OLM when the operator is installed